### ISSUE TITLE
Wipe Into Object Not Working

### ISSUE THREAD

**Issue by vertigo235** [supermerill/SuperSlicer#638]
### Version
Version 2.2.54

### Operating system type + version
Win64

### 3D printer brand / version + firmware version (if known)
Prusa MK3S MMU2S
### Behavior
Wipe into this object does not appear to be working

#### Project File (.3MF) where problem occurs
Added
[wipe_into_object.zip](https://github.com/supermerill/SuperSlicer/files/5497340/wipe_into_object.zip)

**Comment by supermerill** [comment]
What do you mean by "Wipe into this object"?
what makes you believe it's not working? Can you explain me why?

**Comment by vertigo235** [comment]
Actually it might be working, but for whatever reason this group of objects just doesn't seem to use much of the waste object, hmm.

**Comment by vertigo235** [comment]
Seems to only use it for one color that is on every layer, I guess this is working normally.  

Might just need to close this I guess.

**Comment by vertigo235** [comment]
Yeah seems to be a suboptimal behavior I guess, even if you add multiple objects (such as 1 per color that needs to be purged it only seems to use one color per layer into the purge object. Would be nice if it would use a different color on each object to minimize the waste.

**Comment by vertigo235** [comment]
Here is a project with 4 purge objects, but it seems to purge the white color in all 4 objects at each layer. 

[wipe_into_object_multiple_purge_objects.zip](https://github.com/supermerill/SuperSlicer/files/5498222/wipe_into_object_multiple_purge_objects.zip)
